config/plop.config.js

```diff
@@ -0,0 +1,86 @@
+const inquirer = require('inquirer-directory');
+
+function required(name) {
+
+  return (value) => {
+    if (!value) {
+      return `${name} is required`;
+    }
+
+    return true;
+  }
+}
+
+module.exports = (plop) => {
+
+  plop.addPrompt('directory', inquirer);
+
+  plop.setGenerator('component', {
+    description: 'Create a new component',
+    prompts: [
+      {
+        type: 'input',
+        name: 'name',
+        message: 'What is name of new component?',
+        validate: required('name'),
+      },
+      {
+        type: 'confirm',
+        name: 'needPath',
+        message: 'Should I place new component in components directory, or it should be placed somewhere else?',
```

Comment:
I'd recommend `Do you want to specify the path manually (otherwise component will be placed in \`components\`) `